app/controllers/merchant_invoices_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MerchantInvoicesController < ApplicationController
+
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @merchant_invoices = @merchant.invoices.distinct
+  end
+
+  def show
+    # require 'pry'; binding.pry
```

Comment:
I don't know if you meant to leave this in, but there's a pry in the merchant_invoices_controller